src/main/java/com/google/javascript/gents/TypeScriptGenerator.java

```diff
@@ -144,7 +144,10 @@ private void generateTypeScript() {
       }
     }
     try {
-      Files.write(result.moduleRewriteLog, new File(opts.moduleRewriteLog), UTF_8);
+      if (opts.moduleRewriteLog != null) {
+        Files.write(result.moduleRewriteLog, new File(opts.moduleRewriteLog), UTF_8);
+      }
+      // Skip writing if moduleRewriteLog is not used, instead of trying to guess a place to write to.
```

Comment:
I think the comment isn't necessary.  If the user doesn't say where to write an extra file to, it's usually implied it won't be written.

Reply:
I am just never sure what are the expectations with CLI tools. Removed